#please refer to VisualServoMetaType.h
bool RobotAllRight
bool RobotSwitchOn